test/Microsoft.AspNetCore.Razor.Evolution.Test/TestFiles/IntegrationTests/RuntimeCodeGenerationIntegrationTest/ComplexTagHelpers.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@addTagHelper something, nice
+
+@if (true)
+{
+    var checkbox = "checkbox";
+
+    <div class="randomNonTagHelperAttribute">
+        <p time="Current Time: @DateTime.Now">
+            <h1>Set Time:</h1>
+            @if (false)
+            {
+                <p>New Time: <input type="text" value="" placeholder="Enter in a new time..."/></p>
+            }
+            else
+            {
+                <p>Current Time: <input type=@checkbox checked=true /></p>
+                <input tYPe='@(true ? "checkbox" : "anything")' />
+                <input type='@if(true) { <text>checkbox</text> } else { <text>anything</text> }'>
+            }
+        </p>
+        <p unbound="first value" age="@DateTimeOffset.Now.Year - 1970" unbound="second value" >
+            @{ var @object = false;}
+            <input ChecKED="@(@object)">
```

Comment:
This doesn't look to be rendering correctly compared to old world. Also talked offline about combing through a few of these complex ones just to ensure things like this aren't missed.